scripts-python/case.build

```diff
@@ -0,0 +1,508 @@
+#!/usr/bin/env python
+
+"""
```

Comment:
The implementation of this file follows the perl very closely. We can clean up more later.